src/protocol.ts

```diff
@@ -49,394 +52,422 @@ export interface UpdateBufferRequest extends Request {
-	}
-
-	export interface GetCodeActionsRequest extends Request {
-		Selection: Range;
-	}
-
-	export interface OmniSharpCodeAction {
-		Identifier: string;
-		Name: string;
-	}
-
-	export interface GetCodeActionsResponse {
-		CodeActions: OmniSharpCodeAction[];
-	}
-
-	export interface RunCodeActionRequest extends Request {
-		Identifier: string;
-		Selection: Range;
-		WantsTextChanges: boolean;
-	}
-
-	export interface RunCodeActionResponse {
-		Changes: ModifiedFileResponse[];
-	}
-
-
-	export interface MSBuildProjectDiagnostics {
-		FileName: string;
-		Warnings: MSBuildDiagnosticsMessage[];
-		Errors: MSBuildDiagnosticsMessage[];
-	}
-
-	export interface MSBuildDiagnosticsMessage {
-		LogLevel: string;
-		FileName: string;
-		Text: string;
-		StartLine: number;
-		StartColumn: number;
-		EndLine: number;
-		EndColumn: number;
-	}
-
-	export interface ErrorMessage {
-		Text: string;
-		FileName: string;
-		Line: number;
-		Column: number;
-	}
-
-	export interface PackageRestoreMessage {
-		FileName: string;
-		Succeeded: boolean;
-	}
-
-	export interface UnresolvedDependenciesMessage {
-		FileName: string;
-		UnresolvedDependencies: PackageDependency[];
-	}
-
-	export interface PackageDependency {
-		Name: string;
-		Version: string;
-	}
+    export interface Point {
+        Line: number;
+        Column: number;
+    }
+
+    export interface Range {
+        Start: Point;
+        End: Point;
+    }
+
+    export interface GetCodeActionsRequest extends Request {
+        Selection: Range;
+    }
+
+    export interface OmniSharpCodeAction {
+        Identifier: string;
+        Name: string;
+    }
+
+    export interface GetCodeActionsResponse {
+        CodeActions: OmniSharpCodeAction[];
+    }
+
+    export interface RunCodeActionRequest extends Request {
+        Identifier: string;
+        Selection: Range;
+        WantsTextChanges: boolean;
+    }
+
+    export interface RunCodeActionResponse {
+        Changes: ModifiedFileResponse[];
+    }
+
+
+    export interface MSBuildProjectDiagnostics {
+        FileName: string;
+        Warnings: MSBuildDiagnosticsMessage[];
+        Errors: MSBuildDiagnosticsMessage[];
+    }
+
+    export interface MSBuildDiagnosticsMessage {
+        LogLevel: string;
+        FileName: string;
+        Text: string;
+        StartLine: number;
+        StartColumn: number;
+        EndLine: number;
+        EndColumn: number;
+    }
+
+    export interface ErrorMessage {
+        Text: string;
+        FileName: string;
+        Line: number;
+        Column: number;
+    }
+
+    export interface PackageRestoreMessage {
+        FileName: string;
+        Succeeded: boolean;
+    }
+
+    export interface UnresolvedDependenciesMessage {
+        FileName: string;
+        UnresolvedDependencies: PackageDependency[];
+    }
+
+    export interface PackageDependency {
+        Name: string;
+        Version: string;
+    }
+}
+
+// dotnet-test endpoints
+
+export interface GetTestStartInfoRequest {
```

Comment:
Hey @troydai -- are these new v2 endpoints? If so, could you include them in the V2 namespace above?